src/Plugin/Field/FieldFormatter/StrawberryPagedFormatter.php

```diff
@@ -322,9 +286,7 @@ public function settingsSummary() {
         case 'metadatadisplayentity':
           $entity = NULL;
           if ($this->getSetting('metadatadisplayentity_source')) {
-            $entity = $this->entityTypeManager->getStorage(
```

Comment:
Nope. the right way here is multiline as it was before. That new line spans too long. Please revert these few lines